src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1254,6 +1254,46 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                     _TryStopAutoScroll(ptr.PointerId());
                 }
             }
+            const auto terminalPos = _GetTerminalPosition(point.Position());
+            if (terminalPos != _lastHoveredCell)
+            {
+                const auto uri = _terminal->GetHyperlinkAtPosition(terminalPos);
+                if (!uri.empty())
+                {
+                    // Update the tooltip with the URI
+                    LinkTip().Content(winrt::box_value(uri));

```

Comment:
Is there a way we can put two textblocks inside the tooltip, one of which is localized to say "ctrl+click to open link" in italic or something?

I'm imagining...

![image](https://user-images.githubusercontent.com/189190/92165627-2fe7b300-edec-11ea-90c4-a315cf0e9662.png)

Reply:
they might have to be `Run`s